pkg/controller/knativeserving/openshift/openshift.go

```diff
@@ -291,6 +414,23 @@ func egress(u *unstructured.Unstructured) error {
 	return nil
 }
 
+func updateIstioConfig(u *unstructured.Unstructured) error {
+	if u.GetKind() == "ConfigMap" && u.GetName() == "config-istio" {
+		istioConfig := &v1.ConfigMap{}
+		if err := scheme.Convert(u, istioConfig, nil); err != nil {
+			return err
+		}
+		if len(istioConfig.Data) > 0 {
```

Comment:
Why the len check here? What are you trying to verify?

Reply:
Len check not required.
Updated